testplan/exporters/testing/base.py

```diff
@@ -167,6 +167,28 @@ def export(self, source):
             tag_dicts=self.cfg.report_tags_all,
             filter_type=self.ALL)
 
+
+class NetworkExporterConfig(ExporterConfig):
+
+    @classmethod
+    def get_options(cls):
+        return {
+            ConfigOption('url', default=None): Or(str, None)
+        }
+
+
+class NetworkExporter(Exporter):
```

Comment:
I guess we could have other class like DBExporter also inherit this NetworkExporter, then we can have SomeDBExporter inherit DBExporter, they all need a URL in config.

Reply:
A URL is specific to HTTP and a small number of related protocols though, other forms of network transfer will require a different address (e.g. exporting to a MySQL DB will not require a URL). I don't think this is the right abstraction.

Couch DB is unique in that it uses HTTP - I think that when we add a Couch DB exporter it can just inherit from HTTPExporter.

Reply:
I agree with Ryan. Let's keep the class hierarchy simply until we really need to branch.